### PR TITLE
signup add awaiting state

### PR DESCRIPTION
**Pull request by EvGen94**
Added hyperlink to Terms of Service
Add RegistrationAwaiting state

### CODE REVIEW

File: lib/blocs/registration_cubit/registration_cubit.dart [hunk 1]
```diff
@@ -17,6 +17,7 @@ class RegistrationCubit extends Cubit<RegistrationState> {
   }
 
   void prepare() async {
+    emit(RegistrationAwaiting());
```
Comment (bobs4462) [inline lib/blocs/registration_cubit/registration_cubit.dart:20]: Why do you need to emit this state here?

**Review by bobs4462 — CHANGES_REQUESTED** [review]
Check the state emission

File: lib/pages/sign_up.dart [hunk 2]
```diff
@@ -106,6 +115,13 @@ class _SignUpState extends State<SignUp> {
       body: SafeArea(
         child: BlocBuilder<RegistrationCubit, RegistrationState>(
           bloc: Get.find<RegistrationCubit>(),
+          buildWhen: (_, currentState) =>
```
Comment (bobs4462) [inline lib/pages/sign_up.dart:118]: Why list all the states? Just remove the buildWhen...

**Review by bobs4462 — APPROVED** [review]

File: lib/pages/sign_up.dart (hunk 2 above)
Comment (bobs4462) [reply lib/pages/sign_up.dart]: Why this one has not been fixed?

Comment (EvGen94) [reply lib/pages/sign_up.dart]: Man, it's fixed it's outdated code

**Review by bobs4462 — APPROVED** [review]

**Pull request merged**